UnitedSets/UI/AppWindows/MainWindow.xaml.EventHandler.Implementation.cs

```diff
@@ -162,9 +162,13 @@ private async Task SafeClose(TabBase tab)
     private partial void OnWindowClosing(AppWindowClosingEventArgs e)
     {
         e.Cancel = true;//as we will just exit if we want to actually close
-        Win32Window.Focus();
-        ClosingFlyout.XamlRoot = Content.XamlRoot;
-        ClosingFlyout.ShowAt((FrameworkElement)Content);
+        if (UnitedSetsApp.Current.Tabs.Count > 1)
+        {
+            Win32Window.Focus();
+            ClosingFlyout.XamlRoot = Content.XamlRoot;
+            ClosingFlyout.ShowAt((FrameworkElement)Content);
+        }else
+            RequestCloseAsync(CloseMode.ReleaseWindow);
```

Comment:
This looks great! However, I will have to find other way to allow quitting application in the "save and close" mode in case settings are modified.